app/pages/project/components/ProjectNavbar/components/ProjectTitle/ProjectTitle.jsx

```diff
@@ -62,22 +74,25 @@ export const StyledUnderReview = styled.small`
 `;
 
 
-function ProjectTitle({ launched, link, title, underReview }) {
+function ProjectTitle({ launched, link, redirect, title, underReview }) {
+  const TitleComponent = (redirect) ? StyledRedirect : StyledLink;
+
   return (
     <ThemeProvider theme={{ mode: 'light' }}>
       <H1>
         {underReview && !launched &&
           <StyledUnderReview>{counterpart('project.nav.underReview')}</StyledUnderReview>}
-        <StyledLink to={link}>
+        <TitleComponent to={link} href={redirect}>
```

Comment:
not wild about having both props when only one is relevant based on `redirect`, as already checking when declaring `TitleComponent`, but had difficulty with other implementations